pyproject.toml

```
@@ -26,7 +26,7 @@ dependencies = [
     "coredis",
     "fhaviary>=0.14.0",  # For multi-image support
     "limits",
-    "litellm>=1.59.3",
+    "litellm<1.60.7",  # Avoid bug https://github.com/BerriAI/litellm/issues/8507
```

```suggestion
    "litellm>=1.59.3,<1.60.7",  # Avoid bug https://github.com/BerriAI/litellm/issues/8507
```

Why ditch the lower bound? We still need it